### ISSUE TITLE
Question about 3D OCC task training

### ISSUE THREAD

**Issue by DanielMing123** [wzzheng/TPVFormer#50]
Hi Author,

For the 3D occupancy prediction task training, do we still need to set ignore_index=0 when initiate the cross_entroy loss function? In the paper, you said "pseudo-per-voxel labels were generated from sparse point cloud by assigning a new label of empty to any voxel that does not contain any point, and we use voxel predictions as input to both lovasz-softmax and cross-entropy losses." Does that mean for the [100,100,8] 3D volume, we set all rest voxel 's label to 0 as empty label? I found the occupied voxels whose label generate from sparse lidar point, only has around 1000~2300 in total. This approach will cause serious class imbalance. Can author provide more detail information about empty voxel label generation for the 3D occupancy prediction task?
